### ISSUE TITLE
Transferring Your Theme to a New Owner

### ISSUE THREAD

**Issue by kafleg** [WordPress/Documentation-Issue-Tracker#391]
_This is a new page to add under the [releasing your theme](https://developer.wordpress.org/themes/releasing-your-theme/) section._ 

### Here is a Draft

Do you want to transfer your theme to the new owner? Or want to know the process of how to transfer a theme from one author to another? 

There might be various reasons to transfer the theme. If you are not able to maintain it properly, then you can transfer the theme who wants to adopt it and maintain it. 

**Comment on the theme ticket [Recommended]**
The best and recommended way is to comment on the theme ticket.  To find the theme ticket, go to [themes trac](https://themes.trac.wordpress.org/report) and click on **{26} My theme uploads**. There you can see all your themes list. Find the ticket of that theme that you want to transfer and comment in the comment box. (The comment box is on the last part of that page, scroll down). 

In the comment section, you need to provide:

- Provide the new owner's username 
- Comment from the current owner 

We highly recommend mentioning the current [team representative](https://make.wordpress.org/themes/handbook/about/members/) in the comment so that the transfer process will be fast.

**Using Email** 
You can also email at _themes@wordpress.org_ by providing the theme name as well as the link and the new owner WordPress.org username. 

**Using Slack**
You can join the #themereview channel on [WordPress slack](https://make.wordpress.org/chat/) and message there. Team representatives of moderators who have access to change the ownership will transfer the theme.

**Comment by kafleg** [comment]
Or I think we can update here in FAQs, https://make.wordpress.org/themes/handbook/faq/#how-do-i-transfer-my-theme-to-another-user

**Comment by carolinan** [comment]
Yes I think it is best to update the FAQ since this is for the theme directory, not themes in general.

**Comment by dknauss** [comment]
Where is the appropriate place to make edits to new drafts like this? (Assuming this is wanted.)

**Comment by kafleg** [comment]
> Where is the appropriate place to make edits to new drafts like this? (Assuming this is wanted.)

You can [create an issue](https://github.com/WordPress/Documentation-Issue-Tracker/issues/new/choose) and make the draft.

**Comment by kafleg** [comment]
Can I replace the text [here](https://make.wordpress.org/themes/handbook/faq/#how-do-i-transfer-my-theme-to-another-user)? 
![image](https://user-images.githubusercontent.com/13062231/200265235-344d2163-5dd9-403e-bafd-0a08df127267.png)

**Comment by carolinan** [comment]
Yes - and replace leader with "representative",

**Comment by carolinan** [comment]
You already have replaced it 👍  -text looks good.

**Comment by kafleg** [comment]
Done: https://make.wordpress.org/themes/handbook/faq/#how-do-i-transfer-my-theme-to-another-user

**Comment by kafleg** [comment]
Also, FAQ from @TeBenachi is added, https://make.wordpress.org/themes/handbook/faq/#can-i-include-custom-controls-or-options-in-my-block-theme

Closing this issue now.
